bin/build_daq

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -e
+
+tag=
+
+if [ "$1" == "dts" ]; then
+  tag="dts"
+fi
+
+if [ -z "$tag" ]; then
```

Comment:
I would default tag to daq at the top, rather than a replace-if-empty conditional

Reply:
Done